### ISSUE TITLE
Add button to preview card from edit card view

### ISSUE THREAD

**Issue by ceribik** [ankidroid/Anki-Android#3819]
It would be nice to have a button added at the top of the edit card view that allows you to preview all the cards of the current note. Currently, you can only preview a card from the card browser (by long pressing).

**Comment by ceribik** [comment]
I just noticed #3405 which seems to imply this used to be a feature but was removed in an update at some point.

Does anyone know if that is true?

**Comment by rainbyte** [comment]
Today I was trying to find this button, and turned out it doesn't exist.

I think this could be useful when writing math cards using latex.

**Comment by mikehardy** [comment]
Odd - this seems like it should exist yes. There is definitely a Preview feature from the CardTemplateEditor but I don't recall seeing one from the NoteEditor either

**Comment by timrae** [comment]
If you go through the issues and PRs related to the preview function,
you'll find a detailed account of this. IIRC it turned out it wasn't
possible to preview a note that didn't exist in libanki without making
large changes to libanki.

Most likely those changes have already been made now with the recent work
@mikehardy did, and this feature could finally be possible?

**Comment by mikehardy** [comment]
I think we'd need to merge #5151 first (and it needs a bit of work but is currently quiet as I'm not doing anything large pending 2.9 stable), but then yes - Preview can absolutely display uncommitted changes now, so will be possible without much effort.

**Comment by timrae** [comment]
There's no need to wait until 2.9 stable, that's what we have a release branch for.

**Comment by mikehardy** [comment]
True enough - and I dig it - I should say there are multiple reasons, one being I'm busy enough at the moment that with available time I'm focusing on user support and issue triage, the other being that I have bad experience with unreleased code and it seems like opening a 2.10-alpha series before 2.9 is actually 2.9.0 is ambitious for a small crew like us, so I'm just laying low for a bit. I'll get that thing done though - it's too close to leave alone forever

**Comment by david-allison-1** [comment]
I'd like to see this after #5151 is in

**Comment by mikehardy** [comment]
If anyone is still watching this issue, this is available in 2.12alpha10 which is processing in google play store now and should release momentarily, or you can try directly in parallel / next to your main install with this package https://github.com/ankidroid/Anki-Android/releases/download/v2.12alpha10/AnkiDroid-2.12alpha10.parallel.A.apk
